packages/govuk-frontend/src/govuk/components/checkboxes/checkboxes.mjs

```diff
@@ -31,16 +31,18 @@ export class Checkboxes extends GOVUKFrontendComponent {
     super()
 
     if (!($module instanceof HTMLElement)) {
-      throw new ElementError(`[data-module="${Checkboxes.moduleName}"]`, {
+      throw new ElementError({
         componentName: 'Checkboxes',
-        element: $module
+        element: $module,
+        identifier: `Root element (\`$module\`)`
```

Comment:
Not a blocker, and we should configure ESLint to do this, but can avoid the escaping?

```suggestion
        identifier: 'Root element (`$module`)'
```